csrankings-c.csv

```diff
@@ -607,7 +607,7 @@ Chittaranjan Mandal,IIT Kharagpur,http://cse.iitkgp.ac.in/~chitta/pubs,jkoZD_MAA
 Chittaranjan Mandal 0001,IIT Kharagpur,http://cse.iitkgp.ac.in/~chitta,jkoZD_MAAAAJ
 Chiu C. Tan 0001,Temple University,https://cis.temple.edu/~cctan,NOSCHOLARPAGE
 Chiu Chiang Tan,Temple University,https://cis.temple.edu/~cctan,NOSCHOLARPAGE
-Chloe Martindale,University of Bristol,https://www.martindale.info,AKpOGBkAAAAJ
+Chloe Martindale,University of Bristol,https://www.martindale.info,NOSCHOLARPAGE

```

Comment:
Restore (correct) Google Scholar ID.

Reply:
Fixed in f4364212b429591d68057d44d7418b70b1a48cc1